### PR TITLE
Add example code for getSessionStatus, deleteSession

### PR DESCRIPTION
**Pull request by jeffreyyun**
https://linear.app/anon-dot-com/issue/ANON-1167/add-into-examples-repo-usage-for-getsessionstatus-deletesession

https://anondotcom.slack.com/archives/C0755M9JPQ8/p1718674056189139?thread_ts=1718673693.280209&cid=C0755M9JPQ8

### CODE REVIEW

File: runtime-sdk-typescript/index.ts [hunk 1]
```diff
@@ -145,13 +145,25 @@ const main = async () => {
...
+  await executeRuntimeScript({
     client,
     account,
     target: { browserContext: browserContext },
     initialUrl: appUrls[account.app],
     run: actions[account.app],
   });
+  const accountInfo = { ownerId: APP_USER_ID, domain: account.app };
+
+  // DEMONSTRATION of `getSessionStatus`, `deleteSession`
```
Comment (rusty-jules) [inline runtime-sdk-typescript/index.ts:157]: DEMONSTRATION feels a bit strong
```suggestion
Demonstration
```

File: runtime-sdk-typescript/index.ts [hunk 2]
```diff
@@ -145,13 +145,25 @@ const main = async () => {
...
+  let sessionStatus;
+  sessionStatus = await client.getSessionStatus({ account: accountInfo });
```
Comment (rusty-jules) [inline runtime-sdk-typescript/index.ts:159]: A bit cleaner to just do 
```suggestion
  let sessionStatus = await client.getSessionStatus({ account: accountInfo });
```
Unbound variables are not desired

File: runtime-sdk-typescript/index.ts [hunk 3]
```diff
@@ -145,13 +145,25 @@ const main = async () => {
...
+  console.log(`Before deleting session, client session status: ${JSON.stringify(sessionStatus)}`);
+
+  await client.deleteSession({ account: accountInfo });
+  console.log(`Session deleted for ${JSON.stringify(accountInfo)}`);
+
+  sessionStatus = await client.getSessionStatus({ account: accountInfo });
+  console.log(`After deleting session, client session status: ${JSON.stringify(sessionStatus)}`);
```
Comment (rusty-jules) [inline runtime-sdk-typescript/index.ts:166]: Are we sure we want this in the runtime sdk `main` example? Won't this delete any session they just uploaded via the link sdk for testing?

**Review by rusty-jules — CHANGES_REQUESTED** [review]

File: runtime-sdk-typescript/index.ts (hunk 1 above)
Comment (jeffreyyun) [reply runtime-sdk-typescript/index.ts:157]: 🔊 🎺 all stand for DEMONSTRATION of our royal magistrate and his DECREE of CAMPAIGNING

File: runtime-sdk-typescript/index.ts (hunk 3 above)
Comment (jeffreyyun) [reply runtime-sdk-typescript/index.ts:166]: Yes, they could just go link it again.  For example, I've kept my `Vue` open the whole time so it's quite fast to re-link.

Alternatively, I can comment this out. That might be better haha, good call

**Pull request merged**